packages/amplify-graphql-http-transformer/src/graphql-http-transformer.ts

```diff
@@ -182,6 +182,15 @@ export class HttpTransformer extends TransformerPluginBase {
     this.directiveList.push(args);
   };
 
+  fieldOfExtendedType = (
+    parent: ObjectTypeDefinitionNode | InterfaceTypeDefinitionNode | ObjectTypeExtensionNode,
```

Comment:
Don't we call this method only for `ObjectTypeExtensionNode` type from transformers? Can we get rid of `ObjectTypeDefinitionNode | InterfaceTypeDefinitionNode` here?

Reply:
D'oh! You are correct, we do. The `@function` and `@auth` transformers have the correct signature -- I forgot to remove the additional types from the `@http` transformer.